web-ui/src/main/resources/catalog/components/viewer/layermanager/partials/layermanageritem.html

```diff
@@ -1,7 +1,7 @@
 <div ng-if="::!layer.get('loading')" class="flex-col">
   <div class="flex-row width-100 flex-align-stretch">
     <div class="flex-col">
-      <input type="checkbox" ng-model="layer.visible" title="{{'layerVisible'|translate}}"/>
+      <input type="checkbox" id="layer-{{$index}}" ng-model="layer.visible" title="{{'layerVisible'|translate}}"/>
```

Comment:
`data-ng-attr-id="layer-{{$index}}` should be used here.
Also for the `title`: `data-ng-attr-title="{{'layerVisible'|translate}}"`
https://docs.angularjs.org/guide/interpolation#-ngattr-for-binding-to-arbitrary-attributes